### PR TITLE
fix(docs): align feature list correctly

### PR DESCRIPTION
**Pull request by donroyco**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix

## Other information

Before:

Screenshot 2020-10-07 at 15 59 50

After:

Screenshot 2020-10-07 at 19 06 29

### CODE REVIEW

File: docs/src/app/pages/app-home/app-home.tsx [hunk 1]
```diff
@@ -22,10 +22,12 @@ export class AppHome {
 
           <section class="overview" padding-top margin-top>
             <p no-margin>
-              <p no-margin>
-                Create a PWA presentation using Web Components, <a href="https://ionicframework.com">Ionic</a> components and HTML or Markdown 🚀 Cherry on the
-                cake 🍒🎂 DeckDeckGo comes with a <a href="https://deckdeckgo.app">Progressive Web App</a> to remote control your slides 📱
-              </p>
+              Create a PWA presentation using Web Components,{' '}
```
Comment (donroyco) [inline docs/src/app/pages/app-home/app-home.tsx:25]: Not sure why this change got in?

Comment (peterpeterparker) [reply docs/src/app/pages/app-home/app-home.tsx:25]: Probably  prettier

**Comment by peterpeterparker** [comment]
Cool!  Thx for the PR @donroyco  👍
